.github/workflows/test_mlagility.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
This MR sounds like a great excuse to start incrementing the version number of mlagility.